tests/components/Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests/MSBuildTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Evaluation;
+
+namespace Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests
+{
+    public static class MSBuildTestHelper
```

Comment:
Could you make this into a class/struct that implements IDisposable and manages the state? Then cleanup/file additions will be really easy.

I'm thinking something along of the lines of:


```cs
public class MSBuildTestBuild : IDisposable // constructor will create a temp directory
{
  public void Add(string fileName, string contents); // add the contents to the directory

  public Microsoft.Build.Evaluation.Project Build(); // Load up the project

  public void Dispose(); // Clean up directory
}